apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1108,6 +1104,19 @@ get_key_block_by_height(micro_block, Config) ->
     ?assertEqual(PrevKeyBlockHeight, maps:get(<<"height">>, Block)),
     ok.
 
+post_key_block(Config) ->
+    post_key_block(?config(current_block_type, Config), Config).
+
+post_key_block(_CurrentBlockType, Config) ->
+    {ok, 200, PendingKeyBlock} = get_key_blocks_pending_sut(),
+    PowEvidence = lists:duplicate(42, 1),
+    Nonce = 1,
+    KeyBlock = PendingKeyBlock#{<<"pow">> => PowEvidence, <<"nonce">> => Nonce},
+    {ok, 400, Error} = post_key_blocks_sut(KeyBlock),
+    %% Block is always rejected - pow and nonce are not correct.
```

Comment:
A test for the positive case (by using the user HTTP API - i.e. fetch pending key block header, find PoW, post mined key block header) would be beneficial for ensuring use case not broken in future changes. May be addressed in following PR/ticket.

Reply:
I see tracked as https://www.pivotaltracker.com/n/projects/2124891/stories/160445250